.test-infra/kubernetes/postgres/postgres-service-for-debezium.yml

```diff
@@ -0,0 +1,59 @@
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: postgres-for-dev
+  labels:
+    name: postgres
+spec:
+  ports:
+    - port: 5432
+  selector:
+    name: postgres
+  type: LoadBalancer
+
+---
+
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: postgres
+spec:
+  replicas: 1
+  selector:
+    name: postgres
+  template:
+    metadata:
+      name: postgres
+      labels:
+        name: postgres
+    spec:
+      containers:
+        - name: postgres
+          image: postgres
+          args: ["-c", "wal_level=logical","-c","max_wal_senders=4","-c","max_replication_slots=4"]
```

Comment:
Why do we need these args? Seems like that is the only difference between this and postgres-for-dev, are we able to just reuse that yml file instead?

Reply:
Hi! We need to use this args to enable Postgres Change Data Capture (CDC).
The reason for having in different files is try to avoid errors with another tests that use postgres-for-dev file and have different files for postgres with CDC disable and enable.

Reply:
> The reason for having in different files is try to avoid errors with another tests that use postgres-for-dev file and have different files for postgres with CDC disable and enable.

Is there a reason we can't just turn on CDC for all tests? I'm not too worried about changes to this file breaking tests since we should catch that with our CI